dlpy/model.py

```diff
@@ -1009,8 +1063,55 @@ def heat_map_analysis(self, data, mask_width=None, mask_height=None,
         Returns
         -------
         :class:`pandas.DataFrame`
+            Contains Columns: ['I__label_', 'P__label_Dolphin', 'P__label_Giraffe', '_filename_0',
```

Comment:
I'm assuming not all returned objects have columns 'P__label_Dolphin' / 'P__label_Giraffe'